### DISCUSSION TITLE
Clear E03 error (Low temperature)

### DISCUSSION THREAD

**Discussion by espenfjo** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#642] [Q&A]
Is there a way to remotely clear the E03 error (low temperature) other than cutting power to the pump (which I cant do remotely)?
Ambient and water temperature at the moment is above 4C

**Comment by visualapproach** [comment]
I don't think so

**Comment by chunkysteveo** [comment]
Nope. The only "hack" way is to remove the inline breaker on the plug and hook up the power to a smart outlet/smart switch. Then you are able to remotely power cycle the pump, and you have (very dangerously!) bypassed the manual re-activation of the breaker.

This is the way... but not the way!
